app/models/invoice.rb

```diff
@@ -13,7 +13,7 @@ class Invoice < ApplicationRecord
   }
 
   def pretty_created_at
-    created_at.strftime("%A, %B %-d, %Y")
```

Comment:
My only hangup is just still the removal of this from the date formatting-- I Slacked you with a screenshot of how it outputs on mine to address your concerns about the spacing. I'm not sure if that resolves it or not, let me know!